src/centos/7/mlnet/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
 RUN yum install -y \
         perl-Data-Dumper
 
+# Sets the library path to pickup openmp
```

Comment:
Why is this needed?

Reply:
Actually this was the primary reason for this PR to add GCC’s based it in PATH (so we can cleanup CLR_{CC, CXX} from pipeline YAML, which we are using in runtime PR as a temporary workaround). While at it, I thought of revving the version from v7 to v9, and that became the title. If @akoeplinger and co. suggest that we should stick to v7, I will remove the rest of the changes except this.

Reply:
But this is just moving this from the bottom of the file to line 6. This was already part of this file. Also, this dockerfile is for the mlnet repo, we don't use this to build the runtime repo.

Reply:
Ah, sorry, I misread the filename (was on my phone and thought it is about PATH export line ..  GH is not showing the next line) 😄 
In the previous CI run, where it tries to compile mlnet with the just-compiled Clang (using GCC 9 devtoolset in the base layer), it failed to locate system libs. The error was:

```sh
    /usr/bin/ld: cannot find -lstdc++
```

I moved this line above in the anticipation, that it will pick up the correct LD_LIBRARY_PATH this time; now waiting for LLVM build to finish to see the effect.

Reply:
Still failed. It is actually failing to build openmp (not mlnet) after compiling llvm project:
```
Step 4/4 : RUN wget http://releases.llvm.org/3.9.1/openmp-3.9.1.src.tar.xz &&     mkdir -p llvm-3.9.1.src/openmp &&     tar -xf openmp-3.9.1.src.tar.xz --strip 1 -C llvm-3.9.1.src/openmp &&     rm openmp-3.9.1.src.tar.xz &&         mkdir llvmbuild &&     cd llvmbuild &&     cmake         -DCMAKE_CXX_COMPILER=/usr/local/bin/clang++         -DCMAKE_C_COMPILER=/usr/local/bin/clang         -DCMAKE_BUILD_TYPE=Release         -DLLVM_LIBDIR_SUFFIX=64        -DLLVM_ENABLE_EH=1         -DLLVM_ENABLE_RTTI=1         ../llvm-3.9.1.src/openmp     &&     make -j $(($(getconf _NPROCESSORS_ONLN)+1)) &&     make install &&     cd .. &&     rm -r llvmbuild &&     rm -r llvm-3.9.1.src
 ---> Running in b487c9579b28
[snip]
    /usr/bin/ld: cannot find -lstdc++
```